rwiki/rwiki-impl/impl/src/sql/oracle/sakai_rwiki.sql

```diff
@@ -2,4 +2,4 @@ create index irwikicurrentcontent_rwi on  rwikicurrentcontent (rwikiid);
 create index irwikihistorycontent_rwi on  rwikihistorycontent (rwikiid); 
 create index irwikipagepresence_sid on  rwikipagepresence (sessionid);
 create index irwikiobject_realm on  rwikiobject (realm);
-
+alter table rwikiobject add groups longtext default
```

Comment:
The conversion scripts has to be in the sakai-reference project, I'll help you on this.